components/layout_2020/geom.rs

```diff
@@ -20,6 +20,8 @@ pub type LengthOrAuto = AutoOr<Length>;
 pub type LengthPercentageOrAuto<'a> = AutoOr<&'a LengthPercentage>;
 
 pub mod flow_relative {
+    use serde::Serialize;
```

Comment:
Let's put this one at the top of the file.

Reply:
```console
error: cannot find derive macro `Serialize` in this scope
  --> components/layout_2020/geom.rs:25:21
   |
25 |     #[derive(Clone, Serialize)]
   |                     ^^^^^^^^^
   |
   = help: consider importing one of these items:
           crate::geom::Serialize
           serde::Serialize

```

Reply:
includes do not go into modules